### ISSUE TITLE
Deprecate `make`-based building of documentation

### ISSUE THREAD

**Issue by jakirkham** [jakirkham/nanshe#232]
Documentation can now be generated `python setup.py build_sphinx`. To generate different types of output one merely needs to write `python setup.py build_sphinx -b <format>`; where, format is the intended output type. This strategy reproduces pretty much anything except for the multistage `latex` outputs. To handle the `latex` to `pdf` case, a `pdf` command line argument is allowed for the format. This results in the `latex` being generated and then the `pdf` generated by `pdflatex` (if available). Even support for cleaning documentation artifacts has been provided. Ultimately, these changes should be refined to builders an the like for PRs to Sphinx.

TL;DR The `Makefile` and `make.bat` have become redundant as `python setup.py build_sphinx` has full functionality and is OS independent. Maintaining these two copies of the documentation build for two OSes allows for divergence and as such is prone to error. We would like to remove them and use `setup.py` directly.

**Comment by jakirkham** [comment]
Info about this method has been removed from the documentation, but the strategy still remains at present.

**Comment by jakirkham** [comment]
As this has basically been deprecated, this will be closed.
